pkgs/applications/editors/idea/default.nix

```diff
@@ -340,12 +340,12 @@ in
 
   datagrip = buildDataGrip rec {
     name = "datagrip-${version}";
-    version = "2016.3.2";
+    version = "2016.3";
```

Comment:
You don't seem to have pushed the fix to the downgrade, might need to git add before amending the commit